src/commonControls/dateRangeControl/DateRangeControl.ts

```diff
@@ -741,7 +741,7 @@ export class DateRangeControl extends ContainerControl implements InteractionMod
 
     // tsDoc - see Control
     async canHandle(input: ControlInput): Promise<boolean> {
-        const customCanHandle = await evaluateCustomHandleFuncs(this, input);
+        const customCanHandle = await evaluateInputHandlers(this, input);
         const builtInCanHandle =
```

Comment:
could we hoist these up as new-style handler definitions?

Reply:
The problem with DateRangeControl being a ContainerControl has multi-level canHandle logic( canHandle custom blocks in containerControl level and canHandle logic flow on its child controls(which updates lastHandlingChild rather than handleFunc) so I couldn't hoist these.
Similarly with current style of NumberControl being multi-level canHandle logic which needs to be refactored.